drivers/net/mlx5/mlx5_vdpa.c

```diff
@@ -341,6 +341,7 @@ create_split_virtq(struct vdpa_priv *priv, int index,
 	MLX5_SET(virtq, virtq, ctrl_mkey, priv->gpa_mkey_index);
 	MLX5_SET(virtq, virtq, umem_id, info->umem_obj->umem_id);
 	MLX5_SET(virtq, virtq, tisn, priv->tis.id);
+	MLX5_SET(virtq, virtq, doorbell_stride_idx, index);
```

Comment:
Ido - we need to update the PRM here.

this is no longer the stride index, rather the queue index.

Reply:
right, added to the PRM draft changes we send to Ahmad on Sunday.